apps/nextjs-example/src/components/transactionFlows/MultiAgent.tsx

```diff
@@ -14,6 +17,8 @@ import { useToast } from "../ui/use-toast";
 import { LabelValueGrid } from "../LabelValueGrid";
 
 const APTOS_COIN = "0x1::aptos_coin::AptosCoin";
+const TRANSFER_SCRIPT =
+  "0xa11ceb0b0600000006010002030206040802050a10071a1d0837200000000103010100000204060c060c05030001090003060c05030d6170746f735f6163636f756e740e7472616e736665725f636f696e73000000000000000000000000000000000000000000000000000000000000000101000001050b010b020b03380002";
```

Comment:
Could we leave a comment with where/what script this is?

Reply:
That’s totally reasonable.